docs/blog/newsletter-may-2019.rst

```diff
@@ -0,0 +1,130 @@
+* Use services like `MongoDB Stitch <https://www.mongodb.com/cloud/stitch>`_ to push docs feedback directly to a Slack channel.
+* Allow users to directly log Salesforce cases/GitHub issues/Jira tickets.
+* Contact users one-on-one to ask for feedback or arrange to receive regular ongoing feedback. Don't forget to close the loop: let these users know when you improve something based on their feedback.
+
+The two biggest challenges were prioritizing feedback-based improvements without compromising ongoing documentation efforts, and finding the time to make use of the collected feedback. The main strategy people used for addressing these challenges was holding document improvement sessions to discuss and prioritize the backlog.
+
+And of course, some WTD talks have covered user feedback too:
+
+* `CSAT - what’s that? </videos/na/2016/csat-what-s-that-betsy-roseberg/>`_
+* `Feedback handling, community wrangling, panhandling </videos/eu/2016/feedback-handling-community-wrangling-panhandling-chris-mills/>`_
+* `You have already succeeded 2.0 </videos/eu/2017/you-have-already-succeeded-design-critique-guidelines-make-feedback-easier-2-0-christy-lutz/>`_
+
+------------------------------------
+Tips and tricks for producing videos
+------------------------------------
+
+Videos can be a great way of educating users, but it's not always easy to do them well, especially if you haven't worked on video before. Here are some tips from a recent community community:
+
+* Keep in mind that videos are not a "quick project": they can be very time consuming.
+* Create a pipeline for video creation and production that suits your organization's specific needs.
+* Consider developing a library of assets for videos and commit to maintaining it.
+* Be sure to take up-front expenses into account, such as editing software or a higher quality microphone.
+* If you don't have the resources to produce full-fledged tutorial videos, consider shorter videos (five minutes or less) that serve as an introduction to draw the audience into the documentation.
+* Collaborate with other teams! UX, Customer Support, Training, and other teams can all be valuable resources for videos.
+* Many community members suggested that docs teams should only commit to video content if they have a dedicated person, or even team, to work on videos.
+
+If you've got any more questions, head on over to `#video <https://writethedocs.slack.com/messages/video/>`_!
+
+------------------------------------------------------
+The variable fortunes of variable placeholders in docs
+------------------------------------------------------
+
+In code examples, you often need to put in placeholders for values that users must provide. But no matter what a hapless writer tries, it seems inevitable that users won't realise they need to substitute in their own data. Here's a summary of the options that our documentarians have provided for their readers:
```

Comment:
I think "in" is redundant with "substitute" -- "...substitute their own data."

Reply:
agreed, taken it out